connectors/utils.py

```diff
@@ -300,3 +300,31 @@ async def _wait_for_room(self, item):
     async def put(self, item):
         item_size = await self._wait_for_room(item)
         return await super().put((item_size, item))
+
+
+class ConcurrentRunner:
+    def __init__(self, max_concurrency=5, results_cb=None):
+        self.max_concurrency = 5
+        self.tasks = []
+        self.results_cb = results_cb
+
+    def __len__(self):
+        return len(self.tasks)
+
+    def _cb(self, fut):
```

Comment:
Does `cb` mean `callback`? What is `fut`?

Can we use full name instead of abbreviation? It will make the code more readable.

Or is it a convention of Python? I found the param was named `coro` instead of `coroutine` in the official asyncio doc. TBH, I can't link `coro` to `coroutine` if I have no context.

Reply:
`fut` means `future`, as this is the object type here. But you are right, I should use better names, thanks